lib/DashPlatformProtocol.js

```diff
@@ -54,11 +54,13 @@ class DashPlatformProtocol {
       }
 
       this.dataContract = new DataContractFacade(
+        this,
         this.jsonSchemaValidator,
```

Comment:
Does it make sense to pass `jsonSchemaValidator` and `stateRepository` separately since we can use getters inside?